peak2bw.py

```diff
@@ -70,19 +70,18 @@ def main():
 
     # Read input files
     _logger.info('Reading input files')
-    peaks = pd.read_csv(args.input, sep='\t', header=None,
-                        usecols=[0, 1, 2], skiprows=args.skip)
-    sizes = pd.read_csv(args.sizes, sep='\t', header=None)
+    peaks = read_intervals(args.input, skip=args.skip)
+    sizes = read_sizes(args.sizes)
 
     # Subset to peaks in sizes file
     # TODO: Move this test into the df_to_bedGraph function
-    peaks_filtered = peaks[peaks[0].isin(sizes[0])].copy()
-    _logger.info('Retaining ' + str(len(peaks_filtered)) + ' of ' +
-                 str(len(peaks)) + ' peaks in given chromosomes.')
+    peaks_filtered = peaks[peaks['chrom'].isin(sizes['chrom'])].copy()
+    _logger.info('Retaining ' + str(len(peaks_filtered)) + ' of ' + str(len(
+        peaks)) + ' peaks in given chromosomes.')
 
     # Add score of 1 for all peaks
     _logger.info('Adding score')
-    peaks_filtered[3] = 1
+    peaks_filtered['score'] = 1
```

Comment:
do we not need a column of 1st equal to the `len(peaks_filtered)` ?

Reply:
Sorry, I didn't understand this comment. What do you mean by 'column of 1st'?

Reply:
peaks already has 3 columns: 'chrom', 'start' and 'end', and we're adding a 4th column which is 'score'.

Reply:
Sorry, I meant column of ones. `peaks_filtered['score'] = 1` What would be the `len(peaks_filtered['score'] = 1)` does pandas assign it automatically ?

Reply:
Pandas assigns a 1 to the score column in every row of the dataframe.